site/code-snippets/web5/build/decentralized-web-nodes/ update-dwn.js

```diff
@@ -0,0 +1,9 @@
+export async function updateDWN
```

Comment:
```suggestion
export async function updateDwnRecord(web5, createdRecord) {
```

- changed function name to more accurately reflect what it's doing
- added parameter list. in order for this function to be executable, it'll need an instance of web5 and a created record. this needs to be passed to the function by the tests you'll create
- opening curly brace to begin the function's body